src/components/structures/TagPanel.js

```diff
@@ -164,7 +164,7 @@ const TagPanel = createReactClass({
             );
         }
 
-        return <div className={classes}>
+        return <div className={classes} onClick={this.onClearFilterClick}>
             <div className="mx_TagPanel_clearButton_container">
                 { clearButton }
             </div>
```

Comment:
can we get rid of this now then?

Reply:
it looks to be there both intentionally and unintentionally. I've been quietly ignoring it.

Reply:
yea, let's rip it out. It's been disabled for 2 years now: https://github.com/matrix-org/matrix-react-sdk/commit/3bf28a08ec4e46ffe9134c90b533451fce57abd5